include/crocoddyl/multibody/friction-cone.hpp

```diff
@@ -29,30 +29,29 @@ class FrictionConeTpl {
   typedef typename MathBase::Quaternions Quaternions;
 
   explicit FrictionConeTpl();
-  FrictionConeTpl(const Vector3s& normal, const Scalar& mu, std::size_t nf = 4, bool inner_appr = true,
-                  const Scalar& min_nforce = Scalar(0.),
-                  const Scalar& max_nforce = std::numeric_limits<Scalar>::max());
+  FrictionConeTpl(const Vector3s& normal, const Scalar mu, const std::size_t nf = 4, const bool inner_appr = true,
+                  Scalar min_nforce = Scalar(0.), const Scalar max_nforce = std::numeric_limits<Scalar>::max());
   FrictionConeTpl(const FrictionConeTpl<Scalar>& cone);
   ~FrictionConeTpl();
 
-  void update(const Vector3s& normal, const Scalar& mu, bool inner_appr = true, const Scalar& min_nforce = Scalar(0.),
-              const Scalar& max_nforce = std::numeric_limits<Scalar>::max());
+  void update(const Vector3s& normal, Scalar mu, const bool inner_appr = true, const Scalar min_nforce = Scalar(0.),
+              const Scalar max_nforce = std::numeric_limits<Scalar>::max());
 
   const MatrixX3s& get_A() const;
   const VectorXs& get_lb() const;
   const VectorXs& get_ub() const;
   const Vector3s& get_nsurf() const;
-  const Scalar& get_mu() const;
-  const std::size_t& get_nf() const;
-  const bool& get_inner_appr() const;
-  const Scalar& get_min_nforce() const;
-  const Scalar& get_max_nforce() const;
+  Scalar get_mu() const;
+  std::size_t get_nf() const;
+  bool get_inner_appr() const;
+  Scalar get_min_nforce() const;
+  Scalar get_max_nforce() const;
```

Comment:
I guess you referred only to
```c++
  Scalar get_min_nforce() const;
  Scalar get_max_nforce() const;
```
because the other cases are superfluous.
Indeed, if you add those `const` keyworks you will get a warning message.
See below for a more elaborated explanation:
https://stackoverflow.com/questions/1134237/pedantic-gcc-warning-type-qualifiers-on-function-return-type